zerver/openapi/python_examples.py

```diff
@@ -1097,6 +1097,20 @@ def deregister_queue(client: Client, queue_id: str) -> None:
     validate_against_openapi_schema(result, "/events", "delete", "400")
 
 
+@openapi_test_function("/events:get")
+def get_queue(client: Client, queue_id: str) -> None:
+
+    # {code_example|start}
+    # `call_on_each_message` and `call_on_each_event` will automatically register
+    # a queue for you.
```

Comment:
This feels like a non sequitur with the new placement.  I think this belongs in the description, not the code examples.

Reply:
Seems right. Moved to a Note in description.